packages/variant-theme/src/components/variantThemeTool/variantThemeTool.styles.ts

```diff
@@ -0,0 +1,22 @@
+import { makeStyles, shorthands, tokens } from '@fluentui/react-components';
```

Comment:
when generating new packages it should not contain implementation 

https://github.com/microsoft/fluentui-contrib/blob/main/CONTRIBUTING.md#creating-a-new-package
<img width="456" alt="image" src="https://github.com/microsoft/fluentui-contrib/assets/1223799/b71701e5-5a79-45c8-ac96-7278da9b7d72">


please undo these changes

Reply:
Hi @Hotell , I have removed all these codes, please help to take a look. Thanks